packages/KSelect/KSelect.vue

```diff
@@ -435,7 +435,12 @@ export default {
             this.selectItems[i].selected = false
           }
 
-          this.selectItems[i].key = `${this.selectItems[i].label.replace(' ', '-').replace(/[^a-z0-9-_]/gi, '')}-${i}`
+          if (this.selectItems[i].label) {
+            this.selectItems[i].key = `${this.selectItems[i].label.replace(' ', '-')}-${i}`
+          } else {
+            this.selectItems[i].key = `test-label-${i}`
```

Comment:
Since this key could potentially appear even when not during testing, let's rename it 
```suggestion
            this.selectItems[i].key = `k-select-item-label-${i}`
```